OpenRA.Mods.Common/Traits/AppearsOnRadar.cs

```diff
@@ -42,6 +42,14 @@ public void Created(Actor self)
 		{
 			var color = modifier != null ? modifier.RadarColorOverride(self) : self.Owner.Color.RGB;
```

Comment:
@pchote Can you expand on this a little more please? I am failing to see how I can pass color to the modifier.

Also would I also need to check against UsePlayerStanceColors in disguise.cs in this case?